src/Carousel.jsx

```diff
@@ -0,0 +1,100 @@
+/*
+heart icon in upper right corner to favorite- clicking fills in color
+below heart icon is expand image icon. containers for both are 36x36
+clicking on the image brings it out into a light box
+next/prev buttons are on far left & right side of image, vertically centered
+  circular white buttons with a drop shadow, arrows are black single chevrons (svg can be downloaded from etsy)
+  chevron box is 24x24px, button is 48x48
+  drop shadow slightly darkens on hover
+main image ~ 625px wide
+*/
+
+import React from 'react';
+import './styles/styles.css';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      favorite: false
+    };
```

Comment:
May have to do with future plans, but is state necessary in this component? If the favorite tag is all that lives here, it could potentially move to App. That could also potentially make it easier to have favorites persist alongside the images.

Reply:
On Etsy favoriting is actually for the listing, not the image, so it's outside the scope of this module either way.